### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by r0adra93**
Added your words verbatim to help n00bs!

### CODE REVIEW

**Comment by darksaid98** [comment]
At this point, I think any suggestions to reduce these repeatedly asked questions are welcome. However, I doubt this PR in its current form accomplishes that.

Perhaps renaming the section to "**Installing**" or something of the sort would fit better. The paragraph could also be turned into a simple step-by-step instruction which would rid it of unnecessary (I'm not saying unwarranted) passive aggressiveness. 

Not certain if this will really help the target *offenders* as the new paragraph is below **Introduction** anyways and they seem to be blindly downloading the source code as zip. 

One could even take this a step further and place another link next to the **Documentation** one called **Download** which points to `https://github.com/overextended/oxmysql/releases/latest/download/oxmysql.zip`. Although there is a discussion to be had regarding whether we should encourage people blindly throwing resources into their servers.

**TLDR:**
1. Putting download instructions or a notice in the readme isn't a bad idea.
2. I don't think this in its current form helps with the problem it attempts to solve.

**My sample download instructions would look something like this:**
1. Download from <https://github.com/overextended/oxmysql/releases/latest/download/oxmysql.zip>
2. Follow install instructions at <https://overextended.github.io/docs/oxmysql/> 
3. Read this if you have issues <https://overextended.github.io/docs/oxmysql/Getting%20Started/issues>

**Comment by thelindat** [comment]
People should be referring to the documentation, which has all the information they need. They _don't, and then they post a new "issue" or spam screenshots on Discord of the "no such export" error. No amount of easily available information is going to help _those_ people.

Even just scrolling past the repository files is too much effort for people blindly hitting "Download ZIP".
image

Most of the information in the readme shouldn't even be there but ugh too annoying to deal.

**Comment by r0adra93** [comment]
some people are experienced server administrators yet still new to FiveM and how FiveM functions
however, they don't know that overextended has a website or anything else. They look at the github and look straight at the readme and there's nothing there pointing them in any direction so they are stuck searching and reseraching.

Put some information into the readme because some people do read it!

**Comment by darksaid98** [comment]
> People should be referring to the documentation, which has all the information they need. They _don't, and then they post a new "issue" or spam screenshots on Discord of the "no such export" error. No amount of easily available information is going to help _those_ people.
> 
> Even just scrolling past the repository files is too much effort for people blindly hitting "Download ZIP". image
> 
> Most of the information in the readme shouldn't even be there but ugh too annoying to deal.

A **Downloading** section like I described above would be nice, I see how it could help at least a few people checking out the resource for the first time. 

I wouldn't say it's very uncommon to find download information with a link to the documentation for setup/configuration in GitHub readmes. ^_^

**Comment by thelindat** [comment]
> They look at the github and look straight at the readme and there's nothing there pointing them in any direction so they are stuck searching and reseraching.

image
image

Two things in their face immediately, but anyway I changed the format.

> You maybe well versed but someone that has a framework and it states, "update oxmysql" they are going to do just that update oxmysql and usually just grabbing the source works!

image

Gives them a link, even!

---

Anyway, I've removed most of the readme and just have the bare minimum. If they can't figure out how to click a link they have no hope.
